packages/bot/package.json

```diff
@@ -63,6 +65,7 @@
     "eslint-plugin-import": "^2.18.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.14.2",
+    "expose-loader": "^0.7.5",
     "file-saver": "^2.0.2",
     "gulp": "^4.0.2",
```

Comment:
@aaron-binary can you please remove the packages that we are no longer using, like gulp.